packages/core-components/src/components/LogViewer/LogViewerControls.tsx

```diff
@@ -57,7 +60,7 @@ export function LogViewerControls(props: LogViewerControlsProps) {
       <TextField
         size="small"
         variant="standard"
-        placeholder="Search"
+        placeholder={t('logViewer.searchButton.placeholder')}
```

Comment:
Sorry. I was not thinking clearly. This isn't the button, it's the field. So something like `logViewer.searchField.placeholder` instead?